client/modules/Booking/Booking.js

```diff
@@ -156,21 +158,23 @@ export class Booking extends Component {
       startDate: moment(date),
       endDate: moment(date).add(2, "h"),
     });
-    this.expectedCostCal(date, 0);  
+    this.expectedCostCal(date, 0, 0);  
   }
 
   handleEndDateChange(date) {
     this.setState({ endDate: date });
-    this.expectedCostCal(0, date);      
+    this.expectedCostCal(0, date, 0);      
   }
 
-  expectedCostCal(start, end)
+  expectedCostCal(start, end, discount)
   {
```

Comment:
the convention for javascript is to start a code block with the { on the same line as the method or control statement 
ie if(foo) {
...
}
or
myfunc() {
...
}

Reply:
Corrected.